converters/google/convert.go

```diff
@@ -233,13 +234,12 @@ func (c *Converter) addDelete(rc *tfjson.ResourceChange) error {
 		if converter.FetchFullResource == nil || converter.MergeDelete == nil {
 			continue
 		}
-		convertedItems, err := converter.Convert(rd, c.cfg)
-
+		convertedItems, err := convertWrapper(converter, rd, c.cfg)
 		if err != nil {
 			if errors.Cause(err) == resources.ErrNoConversion {
 				continue
 			}
-			return err
+			return fmt.Errorf("failed to convert %s: %s", rd.Kind(), err)
```

Comment:
in terms of letting the user know the address - I think that once the panic is converted to an error, it will already have the address included here: 

- https://github.com/GoogleCloudPlatform/terraform-validator/blob/53973b8b1e9ed2d9b9d41d5dc747bcf40ea3761b/converters/google/convert.go#L203
- https://github.com/GoogleCloudPlatform/terraform-validator/blob/53973b8b1e9ed2d9b9d41d5dc747bcf40ea3761b/converters/google/convert.go#L210

So it should be safe to leave off the Kind, since that is included in the address, and only return the error. That would keep the error message shorter and easier to understand.

Reply:
docs for resource address: https://www.terraform.io/internals/json-format#plan-representation

Reply:
Done.